package.json

```diff
@@ -1,55 +1,60 @@
 {
-  "name": "csh-react-boilerplate",
-  "version": "0.0.1",
-  "private": true,
-  "dependencies": {
-    "@fortawesome/fontawesome-svg-core": "^1.2.19",
-    "@fortawesome/free-brands-svg-icons": "^5.9.0",
-    "@fortawesome/free-solid-svg-icons": "^5.9.0",
-    "@fortawesome/react-fontawesome": "^0.1.4",
-    "connected-react-router": "^5.0.1",
-    "csh-material-bootstrap": "^4.3.1",
-    "cross-fetch": "^3.0.2",
-    "history": "^4.7.2",
-    "oidc-client": "^1.5.4",
-    "prop-types": "^15.6.2",
-    "react": "^16.5.2",
-    "react-dom": "^16.5.2",
-    "react-redux": "^5.0.7",
-    "react-router": "^4.3.1",
-    "react-router-dom": "^4.3.1",
-    "react-scripts": "^3.2.0",
-    "reactstrap": "^7.1.0",
-    "redux": "^4.0.1",
-    "redux-oidc": "^3.1.0"
-  },
-  "scripts": {
-    "start": "react-scripts start",
-    "build": "react-scripts build",
-    "test": "react-scripts test --env=jsdom --passWithNoTests",
-    "eject": "react-scripts eject",
-    "lint": "eslint src/**/*.js src/**/*.jsx"
-  },
-  "devDependencies": {
-    "babel-eslint": "^10.0.2",
-    "eslint": "^6.1.0",
-    "eslint-config-standard": "^13.0.1",
-    "eslint-plugin-import": "^2.18.2",
-    "eslint-plugin-node": "^9.1.0",
-    "eslint-plugin-promise": "^4.2.1",
-    "eslint-plugin-react": "^7.14.3",
-    "eslint-plugin-standard": "^4.0.0"
-  },
-  "browserslist": {
-    "production": [
-      ">0.2%",
-      "not dead",
-      "not op_mini all"
-    ],
-    "development": [
-      "last 1 chrome version",
-      "last 1 firefox version",
-      "last 1 safari version"
-    ]
-  }
+    "name": "csh-react-boilerplate",
+    "version": "1.0.0",
+    "private": true,
+    "dependencies": {
+        "@axa-fr/react-oidc-context": "^3.1.5",
+        "@fortawesome/fontawesome-svg-core": "^1.2.31",
+        "@fortawesome/free-brands-svg-icons": "^5.15.0",
+        "@fortawesome/free-solid-svg-icons": "^5.15.0",
+        "@fortawesome/react-fontawesome": "^0.1.11",
```

Comment:
We don't seem to use any iconography in the example app. Remove these from the default list of dependencies.